src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -61,6 +66,7 @@ IEnumerable<IContentAdminFilter> contentAdminFilters
             _siteService = siteService;
             _contentManager = contentManager;
             _contentDefinitionManager = contentDefinitionManager;
+            _httpContextAccessor = httpContextAccessor;
```

Comment:
You can remove this line, while the only need the `HttpContext`

Reply:
We always use httpContextAccessor to access hHttpContext.

Reply:
But in the context of a controller i think that's okay to use the `HttpContext` property ;)

Reply:
OK, now I get it. Changed.